File: wk07/03-wed/get_squares.rb
def get_squares numbers
  perfect_square = []
  numbers.each do |number|
    number_to_check = Math.sqrt(number)
    if number_to_check % 1 == 0 && !perfect_square.include?(number)
      perfect_square.push(number)
    end
  end
  p perfect_square.sort
end

get_squares([4, 1, 16, 1, 10, 35, 22])

File: wk07/03-wed/getSquares.js
console.log('test');


function userInput(changeInput) {
  var numbersChanged = [];
  var numbers_in = changeInput.split(',');
    for (x=0; x < numbers_in.length; x++) {
      numbersToPush = parseInt(numbers_in[x]);
      numbersChanged.push(numbersToPush);
    }
    return numbersChanged;
}

function getSquares() {
  var changeInput = document.querySelector('.numbers').value;
  var numbers = userInput(changeInput);
  var perfectSquare = [];
  for (i=0; i < numbers.length; i++){
    var numberToCheck = Math.sqrt(numbers[i]);
    if (numberToCheck % 1 === 0 && !perfectSquare.includes(numbers[i])) {
      perfectSquare.push(numbers[i]);
    }
  }
  perfectSquare.sort(function(a,b) {return a-b});
  document.querySelector("Body").append('Result: ' + perfectSquare);
}
